### PR TITLE
Update the documentation of FECollection::find_last...().

### PR DESCRIPTION
**Pull request by bangerth**
This came out of my attempt at understanding #7284. In reference to #7287 and #3511.

### CODE REVIEW

**Comment by bangerth** [comment]
!@#$%^&*

It's fixed now.

File: include/deal.II/hp/fe_collection.h [hunk 1]
```diff
@@ -246,12 +246,27 @@ namespace hp
...
+     * within that set of dominating elements, we find the <i>least</i>
+     * dominant one.
+     *
+     * For example, if
      * an FECollection consists of `{FE_Q(1),FE_Q(2),FE_Q(3),FE_Q(4)}` elements
-     * and we are looking for the least dominant finite element among the
-     * last two elements of this collection (i.e., @p fes is `{2,3}`), then
-     * the answer is FE_Q(3) and therefore this function will return its index
-     * in the FECollection, namely `2`.
+     * and the argument is @p fes equal to `{2,3}`, then the set of dominating
```
Comment (davydden) [inline include/deal.II/hp/fe_collection.h]: ~~is~~ (?)

File: include/deal.II/hp/fe_collection.h [hunk 2]
```diff
@@ -246,12 +246,27 @@ namespace hp
...
+     * elements consists of `{0,1,2}`, of which `2` (i.e., the `FE_Q(3)` is the
```
Comment (davydden) [inline include/deal.II/hp/fe_collection.h]: closing bracket missing

File: include/deal.II/hp/fe_collection.h [hunk 3]
```diff
@@ -246,12 +246,27 @@ namespace hp
...
+     * least dominant one, and then that's what the function returns.
+     *
+     * On the other hand, if the
+     * FECollection consists of
+     * `{FE_Q(1)xFE_Q(1),FE_Q(2)xFE_Q(2),FE_Q(2)xFE_Q(3), FE_Q(3)xFE_Q(2)}`
+     * elements and the argument is again
+     * @p fes equal to `{2,3}`, then the set of dominating
+     * elements consists of `{0,1}` because now neither of the last two
+     * elements dominates the other, of which `1` (i.e., the `FE_Q(2)xFE_Q(2)`
```
Comment (davydden) [inline include/deal.II/hp/fe_collection.h]: closing bracket

**Comment by bangerth** [comment]
Thanks, @davydden, for the careful reading! Updated version now here.

**Comment by davydden** [comment]
since it's documentation only, there is no need to run tests.

**Pull request merged**